code/game/objects/structures/girders.dm

```diff
@@ -89,6 +89,12 @@
 		if (!istype(src.loc, /turf/simulated/floor))
 			usr << "<span class='warning'>A floor must be present to build a false wall!</span>"
 			return
+		if (istype(src.loc, /turf/simulated/wall))
+			usr << "<span class='warning'>There is already a wall present!</span>"
+			return
+		if (locate(/obj/structure/falsewall) in src.loc.contents)
+			usr << "<span class='warning'>There is already a false wall present!</span>"

```

Comment:
A bit of a nitpick that's not your fault but could you change these three uses of 'usr' to 'user', works the same but is a better way to handle it.